src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -52,6 +52,10 @@ object ErrorAggregator {
       "fileLimit",
       descr = "Max number of files to retrieve (batch mode only). Default: All files",
       required = false)
+    val channel: ScallopOption[String] = opt[String](
```

Comment:
This should conflict with `kafkaBroker`, on line 88. Add that and we're good to go!

Reply:
Right on, added it to the `conflicts` list in an amended commit.